ui/pages/Project/components/edit-families-and-individuals/BulkEditForm.jsx

```diff
@@ -112,21 +124,21 @@ const BaseBulkContent = React.memo(({ actionDescription, details, project, name,
           <Table.Body>
             {optionalFields.map(field =>
               <TableRow key={field.header}>
-                <TableCell><BoldText>{field.header}</BoldText></TableCell>
+                <TitleTableCell><BoldText>{field.header}</BoldText></TitleTableCell>
                 <TableCell>{field.description}</TableCell>
               </TableRow>,
             )}
           </Table.Body>
         </StyledTable>
       </div>
-      {details && <div><br />{details}</div>}
-      <br />
     </Container>
+    {details && <div><br />{details}</div>}
     <br />
     <FileUploadField
       clearTimeOut={0}
       dropzoneLabel="Click here to upload a table, or drag-drop it into this box"
-      url={`/api/project/${project.projectGuid}/upload_${name}_table`}
+      url={project ? `/api/project/${project.projectGuid}/upload_${name}_table`
```

Comment:
I would rather have this as an explicitly passed optional url prop rather than hardcoding this somewhat unrelated url in here. So ```url={url || `/api/project/${project.projectGuid}/upload_${name}_table`}```

Reply:
Cool